crates/matrix-sdk/src/client/builder.rs

```diff
@@ -1174,4 +1213,18 @@ pub(crate) mod tests {
             object
         })
     }
+
+    /// The FFI's build has SSS disabled by default, but the SDK has it enabled
+    /// by default. The tests are aligned with the FFI for now (until we remove
+    /// regular sliding sync).
+    fn make_ffi_builder() -> ClientBuilder {
```

Comment:
I wouldn't mention FFI here, because it's not clear for a usual reader what FFI we're referring to, and we need to keep a clear separation of concerns. Maybe rename it `make_non_sss_client_builder`?